pkg/proto/buildqueuestate/buildqueuestate.proto

```diff
@@ -278,6 +278,11 @@ message InvocationState {
   // The total number of client invocations for which one or more
   // operations in the QUEUED execution stage exist.
   uint32 queued_children_count = 9;
+
+  // The total number of operations associated with this platform
+  // queue, invocation and any recursive child invocations that are in
+  // the QUEUED execution stage.
+  uint32 recursive_queued_operations_count = 10;
```

Comment:
Yeah, this is something that BuildQueueState is not great at. I think that for some of these counters we only report the direct value, while others are indeed recursive.

Instead of adding this additional field, what are your thoughts on having some helper type like this:

```proto
message InvocationObjectCount {
  uint32 direct = 1;
  uint32 indirect = 2;
}
```

And that we then change the type of fields like `queued_operations_count` to `InvocationObjectCount`?